config/develop/agora.yaml

```diff
@@ -17,7 +17,7 @@ parameters:
   EbDeploymentPolicy: 'RollingWithAdditionalBatch'
   SnsBounceNotificationEndpoint: 'agora-develop-bounce-notifications@sagebase.org'
   SnsNotificationEndpoint: 'agora-develop@sagebase.org'
-  EbSolutionStackName: '64bit Amazon Linux 2 v5.5.0 running Node.js 12'
+  EbSolutionStackName: '64bit Amazon Linux 2 v5.5.0 running Node.js 16'
```

Comment:
i think we want to make the change to all environments?  if you agree then please update `config/staging/agora.yaml` and `config/prod/agora.yaml` as well.